express-crud/server.ts

```diff
@@ -0,0 +1,147 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.use(express.json()); // Mounts the json middleware, if a req comes in with json it will parse
+
+// returns all rows from the grades table
+app.get('/api/grades', async (req, res, next) => {
+  try {
+    const sql = `
+    select * from "grades";
+    `;
+    const result = await db.query(sql);
+    if (result.rows.length === 0) {
```

Comment:
This is nice, but if you just return `result.rows`, it will return an empty array. So this check isn't needed.